src/frontend/src/functions/urls.tsx

```diff
@@ -1,18 +1,30 @@
 import { ModelInformationDict } from '../components/render/ModelType';
 import { ModelType } from '../enums/ModelType';
+import { base_url } from '../main';
 
 /**
  * Returns the detail view URL for a given model type
  */
-export function getDetailUrl(model: ModelType, pk: number | string): string {
+export function getDetailUrl(
+  model: ModelType,
+  pk: number | string,
+  absolute?: boolean
+): string {
   const modelInfo = ModelInformationDict[model];
 
   if (pk === undefined || pk === null) {
     return '';
   }
 
   if (!!pk && modelInfo && modelInfo.url_detail) {
-    return modelInfo.url_detail.replace(':pk', pk.toString());
+    let url = modelInfo.url_detail.replace(':pk', pk.toString());
+    let base = base_url;
```

Comment:
Question: Is there a reason those are let and not const?

Reply:
Fixed